linux/README.md

```diff
@@ -85,7 +85,7 @@ export _JAVA_OPTIONS="-Xmx4g"
 - replace `<vendor>` with `temurin|dragonwell`
 
 ```shell
-export _JAVA_OPTIONS="-Xmx4g"
```

Comment:
Has the need for 4G gone away?  IIRC you need that amount of memory for the tool to run.

Reply:
Line 88 and line 90 are the same.
> ```
> export _JAVA_OPTIONS="-Xmx4g"
> export COMPOSE_DOCKER_CLI_BUILD=1
> export _JAVA_OPTIONS="-Xmx4g"
> ./gradlew clean packageJdkSuse checkJdkSuse --parallel -PPRODUCT=<vendor> -PPRODUCT_VERSION=<version>
> ```
https://github.com/adoptium/installer/blob/master/linux/README.md?plain=1#L88-L90